samtranslator/model/api/api_generator.py

```diff
@@ -724,12 +724,11 @@ def _add_auth(self):  # type: ignore[no-untyped-def]
 
         if authorizers:
             swagger_editor.add_authorizers_security_definitions(authorizers)  # type: ignore[no-untyped-call]
-            self._set_default_authorizer(  # type: ignore[no-untyped-call]
+            self._set_default_authorizer(
                 swagger_editor,
                 authorizers,
                 auth_properties.DefaultAuthorizer,
                 auth_properties.AddDefaultAuthorizerToCorsPreflight,
-                auth_properties.Authorizers,
```

Comment:
Why do we take out `auth_properties.Authorizers` here? Don't we need it for the method

Reply:
I have removed all use case of `auth_properties.Authorizers` inside the function. This is actually a duplicate of `authorizers` as `authorizers` build from `auth_properties.Authorizers`